src/core/service.c

```diff
@@ -1263,10 +1262,6 @@ static int service_spawn(
         if (r < 0)
                 return r;
 
-        r = unit_full_printf_strv(UNIT(s), c->argv, &argv);
-        if (r < 0)
```

Comment:
Oh, OK, the `%c`-specifier is broken. What about other specifiers?

> If we deprecate %c, I wonder if we could come up with a nice scheme to replace it

I think we don't need a nice scheme to replace it.